src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -438,7 +412,7 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
     }
 
     private subscribeSwitchSite(): void {
-        this.siteService.switchSite$.pipe(takeUntil(this.destroy$)).subscribe(() => {
+        this.siteService.switchSite$.pipe(skip(1), takeUntil(this.destroy$)).subscribe(() => {
```

Comment:
why ?

Reply:
Because `siteService` emits a value the first time when the page loads and it triggers a reload in the page that we should not do.